lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -372,4 +453,4 @@ def print_progress(self, count, intervals, verbose):
                 )
                 start = datetime.now(UTC)
             else:
-                sys.stdout.write('.')
+                sys.stdout.write('.')
```

Comment:
Al final del archivo siempre dejamos una linea vacia, pero no encontré si era un pep8, o solo una costumbre. Tampoco queda claro aquí: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file